packages/core/src/tools/instrumentMethod.ts

```diff
@@ -116,6 +99,7 @@ function createInstrumentedMethod<TARGET extends { [key: string]: any }, METHOD
         onPostCall: (callback) => {
           postCallCallback = callback
         },
+        handlingStack: createHandlingStack(2),
```

Comment:
**💭 thought:** ‏Always computing the stack is useful, it makes the API simple to use. But I wonder if there is a performance cost in generating such stack when we don't use it?